.github/workflows/docs.yml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
I know there's no linting of these files at the moment but this is something `yamllint` always complains about. Do you have plans of running it against these files? I see it's installed during CI runs.

Reply:
Good idea, will add that to the linting job.